assets/app/views/directives/environment.html

```diff
@@ -3,10 +3,11 @@
     <tbody>
       <tr ng-repeat="env in envVars">
         <td>{{env.name}}</td>
-        <td>
+        <!-- Avoid awkward wrapping of long, unbroken values like PATH. -->
+        <td style="word-break: break-all;">
```

Comment:
yeah looks like IE didnt support it till Edge http://www.w3schools.com/cssref/css3_pr_word-break.asp